src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -1,29 +1,60 @@
 ﻿namespace Microsoft.ApplicationInsights.AspNetCore.Extensions
 {
+    using System.Reflection;
+    using Microsoft.Extensions.Logging;
+
     /// <summary>
     /// Application Insights service options defines the custom behavior of the features to add, as opposed to the default selection of featuers obtained from Application Insights.
     /// </summary>
     public class ApplicationInsightsServiceOptions
     {
         /// <summary>
+        /// Initializes a new instance of the <see cref="ApplicationInsightsServiceOptions" /> class.
         /// Application Insights service options that controlls the default behavior of application insights features.
         /// </summary>
         public ApplicationInsightsServiceOptions()
         {
             this.EnableQuickPulseMetricStream = true;
             this.EnableAdaptiveSampling = true;
+            this.LoggerMinimumLevel = LogLevel.Information;
```

Comment:
How does a user control whether or not they want the AI Logger?
How do we control which messages get sent to AI so that users don't accidentally send a lot of data they didn't expect but have to pay for?